packages/core/strapi/lib/types/__tests__/utils/string.js

```diff
@@ -0,0 +1,107 @@
+'use strict';
+
+const path = require('path');
+const { fromFile, t } = require('ts-zen');
+
+const STRING_UTILS_DTS_PATH = path.join(__dirname, '..', 'definitions', 'utils', 'string.d.ts');
```

Comment:
Any workaround for __dirname here so we could just pass a relative path? Something like `fromFile(relPath('../definitions/utils/string.d.ts'))` since this will be at the top of every ts test file and doesn't feel the cleanest.

Reply:
Yeah, that sounds reasonable. Where would you put that though? In Strapi utils? In test utils? In the external lib?